src/components/card-grid.js

```diff
@@ -139,24 +141,37 @@ const CardGrid = (props) => {
     const gridRow = prefixFilteredCourses.slice(i, i + colSize);
     gridArray.push(gridRow);
   }
-  return gridArray.map((row, index) => {
-    return (
-      <Grid.Row key={index} stretched>
-        {row.map((course) => (
-          <Grid.Column key={course.id} columns='equal'>
-            <CourseReviewCard
-              code={course.courseCode}
-              name={course.title}
-              numReviews={course.reviews.length}
-              overallRating={getOverallRating(course)}
-              studyLevel={course.studyLevel}
-              terms={course.terms}
-            />
-          </Grid.Column>))}
-      </Grid.Row>
-    );
-  });
+  console.log(prefixFilteredCourses);
+  if (prefixFilteredCourses.length != 0) {
```

Comment:
okay i wasnt gonna say anything but while you're editing can you restructure so that its:

```
if (prefixFilteredCourses.length == 0) return <NoResultsFound>;

return gridArray.map(....);
```

basically get rid of `else`